target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -138,11 +137,6 @@ ipq40xx_setup_interfaces()
 			"0u@eth0" "2:lan" "3:lan" "4:lan"
 		ucidef_set_interface_wan "eth1"
 		;;
-	pakedge,wr-1)
```

Comment:
not part of this commit?

Reply:
sorry i've updated my local patch using new file, it's fixed now